quint/src/flattening.ts

```diff
@@ -504,6 +285,7 @@ function resolveNamesOrThrow(currentTable: LookupTable, sourceMap: Map<bigint, L
       // This should not happen, as the flattening should not introduce any
       // errors, since parsePhase3 analysis of the original modules has already
       // assured all names are correct.
+      console.log(moduleToString(module))
```

Comment:
A bit of both, I used it for debugging and, since this should never happen (unless I'm modifying the code and break something while doing it), I left it there for my future self.

However, this code is going away when the new flattener comes, so I don't think it is going to be useful again :sweat_smile: